### PR TITLE
Remove async getSubFrames and move sendToFrame loop

### PR DESCRIPTION
**Pull request by Nokel81**
- getSubFrames never needed to be async and was needlessly converting it to an array

- instead of relying on a bunch of promises that will get executed sometime in the future (after the current function ends). Which is a strange and unexpected use of the JS runtime behaviour. This makes it more explicit.

Signed-off-by: Sebastian Malton <sebastian@malton.name>

I have done some testing to make sure that this still works. Namely, I have made sure that adding and removing clusters still works.

### CODE REVIEW

File: src/common/ipc.ts [hunk 1]
```diff
@@ -14,38 +14,31 @@ export async function requestMain(channel: string, ...args: any[]) {
...
+    const type = view.getType();
+
+    logger.silly(`[IPC]: broadcasting "${channel}" to ${type}=${view.id}`, { args });
+    view.send(channel, ...args);
+
+    try {
+      for (const frameInfo of clusterFrameMap.values()) {
+        view.sendToFrame([frameInfo.processId, frameInfo.frameId], channel, ...args);
+      }
+    } catch (error) {
```
Comment (jim-docker) [inline src/common/ipc.ts:38]: this gives up trying to send messages after the first error (I guess that was the old behaviour? Maybe not quite?). Should the `try`/`catch` be inside the `for ()` ?

Comment (Nokel81) [reply src/common/ipc.ts:38]: Might be a good idea. Also need to rebase this so one moment.

Comment (jakolehm) [reply src/common/ipc.ts:38]: I think this functionality was included in #1957.

Comment (Nokel81) [reply src/common/ipc.ts:38]: It was, closing this PR

**Pull request closed without merging**